app-modules/meeting-center/database/migrations/2023_10_16_134402_create_calendar_events_table.php

```diff
@@ -12,6 +12,7 @@ public function up(): void
 
             $table->string('title');
             $table->text('description')->nullable();
+            $table->json('emails')->nullable();
```

Comment:
What do you think about calling this column `attendees` to match the frontend/context?